.github/workflows/patch_release.yml

```diff
@@ -0,0 +1,47 @@
+name: Patch Release
+
+on:
+  workflow_dispatch:
+    inputs:
+      branch:
+        description: 'Branch to build off of'
+        required: true
+
+
+jobs:
+  gradle:
+    runs-on: ubuntu-latest
+    permissions:
+      checks: write
+      contents: write
+      packages: write
+      pages: read
```

Comment:
Why does this have `read` permissions but `release.yml` has write permissions?

Reply:
We could probably downgrade the pages permission of `release.yml` until we actually publish documentation updates, but the plan would be that patch releases don't update the docs on GitHub pages, whereas non-patch releases do.